### PR TITLE
[bn] Localized content/bn/includes/task-tutorial-prereqs.md

### PR DESCRIPTION
**Pull request by Arindam200**
Signed-off-by: Arindam Majumder [arindammajumder2020@gmail.com]
Localizes `content/bn/includes/task-tutorial-prereqs.md` to Bengali
This is an initial PR as may need some improvements!
Tracking issue: https://github.com/kubernetes/website/issues/31677

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Please sign CLA

**Comment by rajibmitra** [comment]
@Arindam200 thank you for the PR. 
LGTM but you have to sign the CLA.

**Comment by Arindam200** [comment]
> @Arindam200 thank you for the PR. 
> LGTM but you have to sign the CLA. 
> 

Thanks for mentioning it! I have signed the CLA !

**Comment by Arindam200** [comment]
> @Arindam200 thank you for the PR. 
> LGTM but you have to sign the CLA. 
> 

@Arhell 

I have signed the CLA! 

What do I have to do now?

**Comment by mitul3737** [comment]
> > @Arindam200 thank you for the PR.
> > LGTM but you have to sign the CLA.
> 
> @Arhell
> 
> I have signed the CLA!
> 
> What do I have to do now?

I will review the code now.

File: content/bn/includes/task-tutorial-prereqs.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+আপনার একটি Kubernetes ক্লাস্টার থাকতে হবে এবং আপনার ক্লাস্টারের সাথে যোগাযোগের জন্য kubectl কমান্ড-লাইন টুলটি কনফিগার করা আবশ্যক । কমপক্ষে দুটি নোড সহ একটি ক্লাস্টারে (যা কন্ট্রোল প্লেন হোস্টট হিসাবে কাজ করছে না) এই টিউটোরিয়ালটি চালাতে হবে । আপনার যদি ইতিমধ্যে একটি ক্লাস্টার না থাকে তবে আপনি [minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/) ব্যবহার করে একটি তৈরি করতে পারেনি বা এই  Kubernetes playground-গুলির মধ্যে একটি ব্যবহার করতে পারেন ।
```
Comment (mitul3737) [inline content/bn/includes/task-tutorial-prereqs.md:1]:
```suggestion
আপনার একটি Kubernetes ক্লাস্টার থাকতে হবে এবং আপনার ক্লাস্টারের সাথে যোগাযোগের জন্য kubectl কমান্ড-লাইন টুলটি কনফিগার করা আবশ্যক । কমপক্ষে দুটি নোড সহ একটি ক্লাস্টারে (যা কন্ট্রোল প্লেন হোস্ট হিসাবে কাজ করছে না) এই টিউটোরিয়ালটি চালাতে হবে । আপনার যদি ইতিমধ্যে একটি ক্লাস্টার না থাকে তবে আপনি [minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/) ব্যবহার করে একটি তৈরি করতে পারেনি বা এই  Kubernetes playground-গুলির মধ্যে একটি ব্যবহার করতে পারেন ।
```

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Review by mitul3737 — APPROVED** [review]
LGTM

**Review by RockingSNP — APPROVED** [review]
Awesome Work 👍👍

**Pull request merged**